drivers/ethernet/eth_stm32_hal.c

```diff
@@ -28,22 +28,137 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #include "eth_stm32_hal_priv.h"
 
 #if defined(CONFIG_ETH_STM32_HAL_USE_DTCM_FOR_DMA_BUFFER) && \
-	    !DT_NODE_HAS_STATUS(DT_CHOSEN(zephyr_dtcm), okay)
+			!DT_NODE_HAS_STATUS(DT_CHOSEN(zephyr_dtcm), okay)
```

Comment:
Please avoid changing code when this does not relate directly to your change. This make change harder to review.
If clean up is required, please do it in a separate commit.
Though, in this case, indentation was correct

Reply:
Ok. I understand this way. Will be changed with next commit.